### DISCUSSION TITLE
Github Desktop is locked up - how do I unlock it?

### DISCUSSION THREAD

**Discussion by boylesg** [community/community#134277] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For god sake!

I cannot get passsed this bloody push request!
Github Desktop won't let it go.
And I can't complete it because it involved a file greater than 100MB - it fails.
There is no obvious way to dump this push request in github desktop.
And nor can I find it in github in my web browser. So I can;t dump it at that end either.
There must be a way to reset github and stop this nonsense????

![image](https://github.com/user-attachments/assets/46eda16b-ad04-41a0-a82f-a0f141fd5050)

**Comment by steveward** [comment]
@boylesg if you're still having issues with this you will need to remove the large file from your repository prior to pushing changes. You can find steps for how to do that [in this article](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-files-from-a-repositorys-history).
